LEGO1/lego/sources/misc/legowegedge.h

```diff
@@ -0,0 +1,28 @@
+#ifndef __LEGOWEGEDGE_H
+#define __LEGOWEGEDGE_H
+
+#include "legoweedge.h"
+#include "mxgeometry/mxgeometry3d.h"
+#include "mxtypes.h"
```

Comment:
It should not have a dependency on `mxtypes.h`. The lego types from `misc` should be used instead